jenkins_jobs/upload_build_artifacts/script.sh

```diff
@@ -12,16 +12,18 @@ else
 fi
 echo  "{'REPO': '$REPOSITORY_FILE_URL', 'BUILD_TIMESTAMP': '${BUILD_TIMESTAMP}', 'BUILD_LOG': '$BUILD_URL', 'BUILD_ID': $BUILD_JOB_NUMBER, 'BUILD_STATUS': '$BUILD_STATUS'}" > STATUS
 
-alias rsync_upload="rsync -e \
-    'ssh -i ${HOME}/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa' \
-    --verbose --compress --stats --times --chmod=a+rwx,g+wx,o-"
+rsync_upload() {
```

Comment:
Could you document what this function expects as parameter (source file path) and mention that destination is hardcoded?